thanos/README.md

```diff
@@ -2,29 +2,63 @@
+
+The main terminologies used in this document can be found in [core-concepts](https://cloudoperators.github.io/greenhouse/docs/getting-started/core-concepts).
+
+## Overview
+
+Observability is often required for operation and automation of service offerings. To get the insights provided by an application and the container runtime environment, you need telemetry data in the form of _metrics_ or _logs_ sent to backends such as _Prometheus_ or _OpenSearch_. With the **Thanos** plugin you can scale the _metric_ part of the observability stack horizontally by hooking into the Prometheus provided by the **kube-monitoring** plugin and optionally save your data in the object storage.
+
+In general, Thanos is a set of components that can be used to extend the storage and retrieval of metrics in Prometheus. It allows you to store metrics in a remote object store and query them across multiple Prometheus servers and Greenhouse clusters. This Plugin is intended to provide a set of pre-configured Thanos components that enable a proven composition. At the core, a set of Thanos components is installed that adds long-term storage capability to a single **kube-monitoring** Plugin and makes both current and historical data available again via one Thanos Query component.
 
 This plugin deploys the following Thanos components:
 
-* Query
-* Compact
-* Store
-<!--* Query Frontend-->
-<!--* (Ruler)-->
+* [Thanos Query](https://thanos.io/tip/components/query.md/)
+* [Thanos Store](https://thanos.io/tip/components/store.md/)
+* [Thanos Compactor](https://thanos.io/tip/components/compact.md/)
+
+Planned components:
+* [Thanos Ruler](https://thanos.io/tip/components/rule.md/)
+* [Thanos Query Frontend](https://thanos.io/tip/components/query.md/)
+
+This Plugin does **not** deploy the following components:
+* [Thanos Sidecar](https://thanos.io/tip/components/sidecar.md/)
+This component is installed in the [kube-monitoring](https://github.com/cloudoperators/greenhouse-extensions/tree/main/kube-monitoring) plugin.
+
+## Note
+
+It is not meant to be a comprehensive package that covers all scenarios. If you are an expert, feel free to configure the Plugin according to your needs.
+
+Contribution is highly appreciated. If you discover bugs or want to add functionality to the plugin, then pull requests are always welcome.
+
+## Quick start
 
-Requirements (detailed steps below):
-* ready to use credentials for a [compatible object store](https://thanos.io/tip/thanos/storage.md/)
-* [thanos-sidecar enabled in Prometheus](#kube-monitoring-plugin-enablement) (usually with [Prometheus Operator](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.ThanosSpec)).
+This guide provides a quick and straightforward way to use **Thanos** as a Greenhouse Plugin on your Kubernetes cluster. The guide is meant to build the following setup:
 
-# Owner
+![Thanos Architecture](./img/Thanos-setup-example.png)
```

Comment:
Cool pic.

I would change the wording of `range queries` to `long term queries`.

A range query would also be a query for the last `[1h]` and this would be resolved by the sidecar. We can also be more precise and note `metrics older than 2h` or similar.